src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/AggregatedRssFeedProducer.java

```diff
@@ -100,4 +101,19 @@ private String addUtmComponents(String url) {
             .build();
     }
 
+    private SyndFeed buildFeed(final List<SyndEntry> feedItems, String requestedUrlString) {
+        final SyndLinkImpl feedLink = new SyndLinkImpl();
```

Comment:
This might be a little thing, but you are using implementation type direclty here and below `SyndFeedImpl` is cast to its general interface, so a little inconsistently used here.